sql/5_0_1-to-5_0_2_upgrade.sql

```diff
@@ -86,3 +86,533 @@
+  SELECT `id`, `pid`, `ODDISC`, `OSDISC`, `ODCUP`, `OSCUP`, `ODMACULA`, `OSMACULA`, `ODVESSELS`, `OSVESSELS`, `ODVITREOUS`, `OSVITREOUS`, `ODPERIPH`, `OSPERIPH`, `ODCMT`, `OSCMT`, `RETINA_COMMENTS`, `DIL_RISKS`, `DIL_MEDS`, `WETTYPE`, `ATROPINE`, `CYCLOMYDRIL`, `TROPICAMIDE`, `CYCLOGYL`, `NEO25`
+  from `form_eye_mag`;
+
+INSERT INTO `form_eye_neuro` (`id`, `pid`, `ACT`, `ACT5CCDIST`, `ACT1CCDIST`, `ACT2CCDIST`, `ACT3CCDIST`, `ACT4CCDIST`, `ACT6CCDIST`, `ACT7CCDIST`, `ACT8CCDIST`, `ACT9CCDIST`, `ACT10CCDIST`, `ACT11CCDIST`, `ACT1SCDIST`, `ACT2SCDIST`, `ACT3SCDIST`, `ACT4SCDIST`, `ACT5SCDIST`, `ACT6SCDIST`, `ACT7SCDIST`, `ACT8SCDIST`, `ACT9SCDIST`, `ACT10SCDIST`, `ACT11SCDIST`, `ACT1SCNEAR`, `ACT2SCNEAR`, `ACT3SCNEAR`, `ACT4SCNEAR`, `ACT5CCNEAR`, `ACT6CCNEAR`, `ACT7CCNEAR`, `ACT8CCNEAR`, `ACT9CCNEAR`, `ACT10CCNEAR`, `ACT11CCNEAR`, `ACT5SCNEAR`, `ACT6SCNEAR`, `ACT7SCNEAR`, `ACT8SCNEAR`, `ACT9SCNEAR`, `ACT10SCNEAR`, `ACT11SCNEAR`, `ACT1CCNEAR`, `ACT2CCNEAR`, `ACT3CCNEAR`, `ACT4CCNEAR`, `MOTILITYNORMAL`, `MOTILITY_RS`, `MOTILITY_RI`, `MOTILITY_RR`, `MOTILITY_RL`, `MOTILITY_LS`, `MOTILITY_LI`, `MOTILITY_LR`, `MOTILITY_LL`, `MOTILITY_RRSO`, `MOTILITY_RLSO`, `MOTILITY_RRIO`, `MOTILITY_RLIO`, `MOTILITY_LRSO`, `MOTILITY_LLSO`, `MOTILITY_LRIO`, `MOTILITY_LLIO`, `NEURO_COMMENTS`, `STEREOPSIS`, `ODNPA`, `OSNPA`, `VERTFUSAMPS`, `DIVERGENCEAMPS`, `NPC`, `DACCDIST`, `DACCNEAR`, `CACCDIST`, `CACCNEAR`, `ODCOLOR`, `OSCOLOR`, `ODCOINS`, `OSCOINS`, `ODREDDESAT`, `OSREDDESAT`)
+  SELECT `id`, `pid`, `ACT`, `ACT5CCDIST`, `ACT1CCDIST`, `ACT2CCDIST`, `ACT3CCDIST`, `ACT4CCDIST`, `ACT6CCDIST`, `ACT7CCDIST`, `ACT8CCDIST`, `ACT9CCDIST`, `ACT10CCDIST`, `ACT11CCDIST`, `ACT1SCDIST`, `ACT2SCDIST`, `ACT3SCDIST`, `ACT4SCDIST`, `ACT5SCDIST`, `ACT6SCDIST`, `ACT7SCDIST`, `ACT8SCDIST`, `ACT9SCDIST`, `ACT10SCDIST`, `ACT11SCDIST`, `ACT1SCNEAR`, `ACT2SCNEAR`, `ACT3SCNEAR`, `ACT4SCNEAR`, `ACT5CCNEAR`, `ACT6CCNEAR`, `ACT7CCNEAR`, `ACT8CCNEAR`, `ACT9CCNEAR`, `ACT10CCNEAR`, `ACT11CCNEAR`, `ACT5SCNEAR`, `ACT6SCNEAR`, `ACT7SCNEAR`, `ACT8SCNEAR`, `ACT9SCNEAR`, `ACT10SCNEAR`, `ACT11SCNEAR`, `ACT1CCNEAR`, `ACT2CCNEAR`, `ACT3CCNEAR`, `ACT4CCNEAR`, `MOTILITYNORMAL`, `MOTILITY_RS`, `MOTILITY_RI`, `MOTILITY_RR`, `MOTILITY_RL`, `MOTILITY_LS`, `MOTILITY_LI`, `MOTILITY_LR`, `MOTILITY_LL`, `MOTILITY_RRSO`, `MOTILITY_RLSO`, `MOTILITY_RRIO`, `MOTILITY_RLIO`, `MOTILITY_LRSO`, `MOTILITY_LLSO`, `MOTILITY_LRIO`, `MOTILITY_LLIO`, `NEURO_COMMENTS`, `STEREOPSIS`, `ODNPA`, `OSNPA`, `VERTFUSAMPS`, `DIVERGENCEAMPS`, `NPC`, `DACCDIST`, `DACCNEAR`, `CACCDIST`, `CACCNEAR`, `ODCOLOR`, `OSCOLOR`, `ODCOINS`, `OSCOINS`, `ODREDDESAT`, `OSREDDESAT`
+  from `form_eye_mag`;
+
+INSERT INTO `form_eye_locking` (`id`, `pid`, `IMP`, `PLAN`, `Resource`, `Technician`, `LOCKED`, `LOCKEDDATE`, `LOCKEDBY`)
+  SELECT `id`, `pid`, `IMP`, `PLAN`, `Resource`, `Technician`, `LOCKED`, `LOCKEDDATE`, `LOCKEDBY` FROM `form_eye_mag`;
+
+RENAME TABLE `form_eye_mag` TO `form_eye_mag_bk`;
```

Comment:
one more thing: will be best to completely remove this table. Users should be making a backup before upgrades and it allows OpenEMR to be 100% innodb (ie. that table lingering around may break things in the future),